libs/user/user.ts

```diff
@@ -1,4 +1,4 @@
-import path from "path";
+import path from "node:path";
```

Comment:
Delete `path` import, please double-check but pretty sure that after the `logger` change we no longer need it